src/query/Query.ts

```diff
@@ -394,11 +406,41 @@ export default class Query<T extends Model = Model> {
   }
 
   /**
-   * Check whether the given field and value combination is filterable through
-   * primary key direct look up.
+   * Convert value to string for composite primary keys as it expects an array.
+   * Otherwise return as is.
+   *
+   * Throws an error when malformed value is given:
+   * - Composite primary key defined on model, expects value to be array.
+   * - Normal primary key defined on model, expects a primitive value.
+   */
+  private normalizeIndexId (value: Options.PrimaryKey): string | number {
+    if (this.model.isCompositePrimaryKey()) {
+      if (!Array.isArray(value)) {
+        throw new Error(`
+          [Vuex ORM] Entity \`${this.entity}\` is configured with a composite primary key
+          and expects an array value but instead received: \`${JSON.stringify(value)}\`
+        `)
+      }
+
+      return JSON.stringify(value)
+    }
+
+    if (Array.isArray(value)) {
+      throw new Error(`
```

Comment:
I think this messages has to use single quote strings. Because back tick string will include the indexes as a string and make lot's of spacing before the error messages on browser console. Maybe I'm wrong...?

```suggestion
throw new Error(
  '[Vuex ORM] Entity`' + this.entity + '`expects a single value but' +
  'instead received an array`:' + JSON.stringify(value)
)
```

Reply:
You’re right. This isn’t the case in Chrome or Node but it will affect other browsers. Thanks for pointing this out! I’ll get this resolved.